src/search1_rpc/service.py

```diff
@@ -48,9 +57,10 @@ def search_objects(params, meta):
         params = params[0]
     start = time.time()
     query = convert_params.search_objects(params)
-    result = convert_result.search_objects(params, search(query, meta), meta)
+    search_result = trap_error(lambda: search(query, meta))
```

Comment:
Just so I follow... You are basically trapping the errors so you can construct the proper error response to hand back to sanic versus letting sanics default handler do this since it would return back a JSON RPC 2.0.  Correct?

Reply:
That is correct.